stable/memcached/README.md

```diff
@@ -52,6 +52,7 @@ The following table lists the configurable parameters of the Memcached chart and
 | `metrics.resources`       | CPU/Memory resource requests/limits for the metrics exporter | `{}`                       |
 | `extraContainers`         | Container sidecar definition(s) as string | Un-set                                        |
 | `extraVolumes`            | Volume definitions to add as string | Un-set                                              |
+| `statefulSet`             | Deploy as a StatefulSet, or a Deployment | true                                           |
```

Comment:
I think it is more common to have a value kind that can be either StatefulSet or Deployment. There are some other charts already that allow for this. E.g nginx-ingress can run either as DaemonSet or as Deployment.

Reply:
Thanks! I was actually looking for an example chart that did this - I have set it up like that one.